README.md

```diff
@@ -29,6 +29,15 @@ Run `./build.sh`* to build. The first time you run this, it could take a while,
 
 \* On Windows, it's `.\build.ps1`. Though you may need `powershell.exe -executionpolicy bypass -file .\build.ps1` due to permissions issues. I don't personally know a thing about Windows permissions, so if anyone has a better idea please let me know.
 
+## Cross-compiling
+
+If we're on x86-64 Linux, we can use `nix develop` to enter a shell from which we can cross-compile:
+```sh
+~/.ghcup/bin/runghc Build.hs --target=x86_64-w64-mingw32
```

Comment:
If we just use `nix build .#x86_64-w64-mingw32:monpad:exe:monpad`, we would need to add some Nix config for compiling the non-Haskell parts of the project.

Perhaps it could just shell out to `runghc Build.hs assets`.